inference-engine/tests/unit/cpu/cpu_convert.cpp

```diff
@@ -0,0 +1,145 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <gtest/gtest.h>
+#include <ngraph/type/float16.hpp>
+#include "nodes/common/cpu_convert.h"
+
+using namespace InferenceEngine;
+using namespace ngraph;
+
+using cpuConvertTestParamsSet = std::tuple<Precision,  // input precision
+                                           Precision>; // target precision
+
+class CpuConvertTest : virtual public ::testing::Test, public testing::WithParamInterface<cpuConvertTestParamsSet> {
+public:
+    static std::string getTestCaseName(testing::TestParamInfo<cpuConvertTestParamsSet> obj) {
+        Precision inputPrecision, targetPrecision;
+        std::tie(inputPrecision, targetPrecision) = obj.param;
+
+        std::ostringstream result;
+
+        result << "InputPrecision=" << inputPrecision << "_";
+        result << "TargetPrecision=" << targetPrecision;
+
+        return result.str();
+    }
+
+protected:
+    void SetUp() override {
+        const size_t size = 100;
+        Precision inputPrecision, targetPrecision;
+        std::tie(inputPrecision, targetPrecision) = this->GetParam();
+
+        void *srcData = new uint8_t[size * inputPrecision.size()];
+        void *dstData = new uint8_t[size * targetPrecision.size()];
+        void *refData = new uint8_t[size * targetPrecision.size()];
```

Comment:
Maybe I missed smt but this memory is not freed.

Reply:
changed on ```std::vector```